private_acl_rules.tf

```diff
@@ -139,7 +139,7 @@ resource "aws_network_acl_rule" "private_egress_to_operations_via_vnc" {
   network_acl_id = aws_network_acl.private[each.value].id
   egress         = true
   protocol       = "tcp"
-  rule_number    = 120 + index(var.private_subnet_cidr_blocks, each.value)
+  rule_number    = 130 + index(var.private_subnet_cidr_blocks, each.value)
```

Comment:
🧙 Seems magical.  Is this counting by 10 jazz useful?  Does this break if index ≥ 10?

Reply:
Yes, it breaks if `index >= 10`.  Our numbering of the ACL rules has always sucked.  Do you have any thoughts as to how it can be improved?